src/node/interfaces.cpp

```diff
@@ -540,8 +540,12 @@ class ChainImpl : public Chain
         const CBlockIndex* block2 = m_node.chainman->m_blockman.LookupBlockIndex(block_hash2);
         const CBlockIndex* ancestor = block1 && block2 ? LastCommonAncestor(block1, block2) : nullptr;
         // Using & instead of && below to avoid short circuiting and leaving
-        // output uninitialized.
-        return FillBlock(ancestor, ancestor_out, lock, active) & FillBlock(block1, block1_out, lock, active) & FillBlock(block2, block2_out, lock, active);
+        // output uninitialized. This requires at least one of the boolean
+        // operands to be cast to int to avoid -Wbitwise-instead-of-logical
+        // compiler warnings.
+        return static_cast<int>(FillBlock(ancestor, ancestor_out, lock, active))
```

Comment:
```suggestion
        // output uninitialized. Cast bool to int to avoid -Wbitwise-instead-of-logical
        // compiler warnings.
        return int{FillBlock(ancestor, ancestor_out, lock, active)}
```

Reply:
Thanks--agree, done.

Reply:
One sec, mispushed.

Reply:
Done.